lib/lotus/cli.rb

```diff
@@ -79,6 +79,19 @@ def new(name = nil)
       end
     end
 
+    desc 'generate', 'generates a new action'
+    method_option :path,                desc: 'applications path',        type: :string, default: 'apps'
```

Comment:
I'd vote against letting dev to customise it. Once we get to the point where we allow different architecture, we could always go back to this.